lib/segment/src/vector_storage/mod.rs

```diff
@@ -30,6 +30,6 @@ pub use vector_storage_base::*;
 // - <https://github.com/rust-lang/rust/issues/88581>
 // - <https://doc.rust-lang.org/std/primitive.usize.html#method.div_ceil>
 #[inline]
-const fn div_ceil(a: usize, b: usize) -> usize {
+pub const fn div_ceil(a: usize, b: usize) -> usize {
```

Comment:
To prevent using this _hack_ everywhere, we might want `pub(crate)` instead:

```suggestion
pub(crate) const fn div_ceil(a: usize, b: usize) -> usize {
```